lib/dartdoc.dart

```diff
@@ -252,12 +252,13 @@ class DartDoc {
     while (result.hasMoreWork) {
       result = context.performAnalysisTask();
     }
-
+    
     // Use the includeExternals.
     for (Source source in context.librarySources) {
       LibraryElement library = context.computeLibraryElement(source);
```

Comment:
We can always optimize! Though you will see a slowdown once flutter is back to generating *all* the docs, I saw ~20 sec difference on my machine between the two cases.